library/alloc/src/rc.rs

```diff
@@ -2439,15 +3046,15 @@ impl<T: ?Sized> Weak<T> {
     #[must_use]
     #[stable(feature = "weak_counts", since = "1.41.0")]
     pub fn weak_count(&self) -> usize {
```

Comment:
```suggestion
    #[inline]
    #[must_use]
    #[stable(feature = "weak_counts", since = "1.41.0")]
    pub fn weak_count(&self) -> usize {
```

Reply:
I'm not sure about this. This is unrelated to the PR and may have affected the perf-run, which may compensate for another performance regression.